fvtest/compilerunittest/aarch64/BinaryEncoder.cpp

```diff
@@ -1675,7 +1676,13 @@ INSTANTIATE_TEST_CASE_P(VectorSplatsImm4, ARM64VectorSplatsImmediateEncodingTest
     std::make_tuple(TR::InstOpCode::bad,  TR::RealRegister::v0, TR::Int64,            0x1ff0000000000LL, ""),
     std::make_tuple(TR::InstOpCode::bad,  TR::RealRegister::v0, TR::Int64,           0x55000000000000LL, ""),
     std::make_tuple(TR::InstOpCode::bad,  TR::RealRegister::v0, TR::Int64,          0x1ff000000000000LL, ""),
-    std::make_tuple(TR::InstOpCode::bad,  TR::RealRegister::v0, TR::Int64,         0x5500000000000000LL, "")
+    std::make_tuple(TR::InstOpCode::bad,  TR::RealRegister::v0, TR::Int64,         0x5500000000000000LL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0x00000ff000000ff0LL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0x000ff000000ff000LL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0x00f0000f00f0000fLL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0x0f00f0000f00f000LL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0x0ff000000ff00000LL, ""),
+    std::make_tuple(TR::InstOpCode::bad, TR::RealRegister::v0, TR::Int64,          0xf00ff00ff00ff00fLL, "")
```

Comment:
Please add one more whitespace after `TR::InstOpCode::bad,` and remove one after `TR::Int64,` in these lines.

Reply:
Done!